themes/default/content/docs/intro/console/ci-cd-integration-assistant.md

```diff
@@ -10,7 +10,10 @@ aliases:
 - /docs/intro/console/extensions/ci-cd-integration-assistant/
 ---
 
-The CI/CD integration assistant helps you integrate Pulumi into CI/CD systems for automatically deploying stacks.
+{{% notes "info" %}}
+The CI/CD integration assistant helps you integrate Pulumi into CI/CD systems for automatically deploying stacks and is only available
+on [Organizations](https://www.pulumi.com/docs/intro/console/organizations/), not personal accounts.
```

Comment:
it seems as though notes do not support links, I've opened #207

Reply:
@susanev How do you mean? I see this link in the preview:

![image](https://user-images.githubusercontent.com/274700/117498643-ee854580-af2e-11eb-8e17-39c5b09d8ee0.png)

Specifically it's the wrapping of the `notes` shortcode with `%`s that makes the Markdown rendering work; that tells Hugo to render the content contained as Markdown.

I feel like I might be misunderstanding, though.

Reply:
oh man, what the heck was i even doing this morning, thank you!!